2024-08-09 12:06:57,629 INFO    StreamThr :527930 [internal.py:wandb_internal():86] W&B internal server running at pid: 527930, started at: 2024-08-09 12:06:57.629369
2024-08-09 12:06:57,630 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: status
2024-08-09 12:06:57,631 INFO    WriterThread:527930 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/run-2lwarbf1.wandb
2024-08-09 12:06:57,631 DEBUG   SenderThread:527930 [sender.py:send():382] send: header
2024-08-09 12:06:57,670 DEBUG   SenderThread:527930 [sender.py:send():382] send: run
2024-08-09 12:06:57,887 INFO    SenderThread:527930 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/files
2024-08-09 12:06:57,888 INFO    SenderThread:527930 [sender.py:_start_run_threads():1136] run started: 2lwarbf1 with start time 1723176417.629289
2024-08-09 12:06:57,903 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: check_version
2024-08-09 12:06:57,903 DEBUG   SenderThread:527930 [sender.py:send_request():409] send_request: check_version
2024-08-09 12:06:58,007 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: run_start
2024-08-09 12:06:58,020 DEBUG   HandlerThread:527930 [system_info.py:__init__():32] System info init
2024-08-09 12:06:58,020 DEBUG   HandlerThread:527930 [system_info.py:__init__():47] System info init done
2024-08-09 12:06:58,020 INFO    HandlerThread:527930 [system_monitor.py:start():194] Starting system monitor
2024-08-09 12:06:58,020 INFO    SystemMonitor:527930 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-09 12:06:58,020 INFO    HandlerThread:527930 [system_monitor.py:probe():214] Collecting system info
2024-08-09 12:06:58,022 INFO    SystemMonitor:527930 [interfaces.py:start():190] Started cpu monitoring
2024-08-09 12:06:58,023 INFO    SystemMonitor:527930 [interfaces.py:start():190] Started disk monitoring
2024-08-09 12:06:58,024 INFO    SystemMonitor:527930 [interfaces.py:start():190] Started gpu monitoring
2024-08-09 12:06:58,025 INFO    SystemMonitor:527930 [interfaces.py:start():190] Started memory monitoring
2024-08-09 12:06:58,026 INFO    SystemMonitor:527930 [interfaces.py:start():190] Started network monitoring
2024-08-09 12:06:58,037 DEBUG   HandlerThread:527930 [system_info.py:probe():196] Probing system
2024-08-09 12:06:58,037 DEBUG   HandlerThread:527930 [system_info.py:_probe_git():181] Probing git
2024-08-09 12:06:58,041 DEBUG   HandlerThread:527930 [system_info.py:_probe_git():189] Probing git done
2024-08-09 12:06:58,041 DEBUG   HandlerThread:527930 [system_info.py:probe():244] Probing system done
2024-08-09 12:06:58,041 DEBUG   HandlerThread:527930 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-44-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-09T04:06:58.037076', 'startedAt': '2024-08-09T04:06:57.624718', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/Gemma-2B', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:100000]', '--output_dir', '/home/rwkv/JL/out_model/llama-test', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '1', '--gradient_accumulation_steps', '32', '--save_strategy', 'steps', '--save_steps', '4000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True', '--run_name', 'gbmm'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': '74dce7f004b5b16bfd24c8ca22e737aee6ff0769'}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 1191.0870357142855, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 1490.742, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1419.512, 'min': 800.0, 'max': 5500.0}, {'current': 1118.903, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1144.86, 'min': 800.0, 'max': 5500.0}, {'current': 1297.361, 'min': 800.0, 'max': 5500.0}, {'current': 1418.334, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 1161.727, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 1462.322, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 1731.217, 'min': 800.0, 'max': 4300.0}, {'current': 1382.667, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 1310.34, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 481.33220291137695}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.55397415161133}}
2024-08-09 12:06:58,041 INFO    HandlerThread:527930 [system_monitor.py:probe():224] Finished collecting system info
2024-08-09 12:06:58,041 INFO    HandlerThread:527930 [system_monitor.py:probe():227] Publishing system info
2024-08-09 12:06:58,041 DEBUG   HandlerThread:527930 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-09 12:06:58,042 DEBUG   HandlerThread:527930 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-09 12:06:58,042 DEBUG   HandlerThread:527930 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-09 12:06:58,888 INFO    Thread-12 :527930 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/files/requirements.txt
2024-08-09 12:06:58,888 INFO    Thread-12 :527930 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/files/conda-environment.yaml
2024-08-09 12:06:59,707 DEBUG   HandlerThread:527930 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-09 12:06:59,707 DEBUG   HandlerThread:527930 [system_info.py:_save_code():90] Saving code
2024-08-09 12:06:59,708 DEBUG   HandlerThread:527930 [system_info.py:_save_code():111] Saving code done
2024-08-09 12:06:59,708 DEBUG   HandlerThread:527930 [system_info.py:_save_patches():128] Saving git patches
2024-08-09 12:06:59,746 DEBUG   HandlerThread:527930 [system_info.py:_save_patches():170] Saving git patches done
2024-08-09 12:06:59,746 INFO    HandlerThread:527930 [system_monitor.py:probe():229] Finished publishing system info
2024-08-09 12:06:59,747 DEBUG   SenderThread:527930 [sender.py:send():382] send: files
2024-08-09 12:06:59,747 INFO    SenderThread:527930 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-09 12:06:59,748 INFO    SenderThread:527930 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-09 12:06:59,748 INFO    SenderThread:527930 [sender.py:_save_file():1392] saving file diff.patch with policy now
2024-08-09 12:06:59,751 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: stop_status
2024-08-09 12:06:59,751 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-09 12:06:59,751 DEBUG   SenderThread:527930 [sender.py:send_request():409] send_request: stop_status
2024-08-09 12:06:59,887 INFO    Thread-12 :527930 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/files/conda-environment.yaml
2024-08-09 12:06:59,887 INFO    Thread-12 :527930 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/files/code/llama/gmm.py
2024-08-09 12:06:59,887 INFO    Thread-12 :527930 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/files/diff.patch
2024-08-09 12:06:59,887 INFO    Thread-12 :527930 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/files/wandb-metadata.json
2024-08-09 12:06:59,888 INFO    Thread-12 :527930 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/files/code
2024-08-09 12:06:59,888 INFO    Thread-12 :527930 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/files/code/llama
2024-08-09 12:06:59,979 DEBUG   SenderThread:527930 [sender.py:send():382] send: telemetry
2024-08-09 12:06:59,979 DEBUG   SenderThread:527930 [sender.py:send():382] send: config
2024-08-09 12:06:59,979 DEBUG   SenderThread:527930 [sender.py:send():382] send: metric
2024-08-09 12:06:59,979 DEBUG   SenderThread:527930 [sender.py:send():382] send: telemetry
2024-08-09 12:06:59,979 DEBUG   SenderThread:527930 [sender.py:send():382] send: metric
2024-08-09 12:06:59,979 WARNING SenderThread:527930 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-09 12:06:59,979 DEBUG   SenderThread:527930 [sender.py:send():382] send: telemetry
2024-08-09 12:06:59,979 DEBUG   SenderThread:527930 [sender.py:send():382] send: telemetry
2024-08-09 12:06:59,979 DEBUG   SenderThread:527930 [sender.py:send():382] send: config
2024-08-09 12:07:00,011 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2
2024-08-09 12:07:00,011 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5251', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:00,011 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3D2FE00AE0C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:00,030 DEBUG   SenderThread:527930 [sender.py:send():382] send: exit
2024-08-09 12:07:00,030 INFO    SenderThread:527930 [sender.py:send_exit():589] handling exit code: 1
2024-08-09 12:07:00,030 INFO    SenderThread:527930 [sender.py:send_exit():591] handling runtime: 2
2024-08-09 12:07:00,030 INFO    SenderThread:527930 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-09 12:07:00,030 INFO    SenderThread:527930 [sender.py:send_exit():597] send defer
2024-08-09 12:07:00,031 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:07:00,031 INFO    HandlerThread:527930 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-09 12:07:00,031 DEBUG   SenderThread:527930 [sender.py:send_request():409] send_request: defer
2024-08-09 12:07:00,031 INFO    SenderThread:527930 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-09 12:07:00,031 INFO    SenderThread:527930 [sender.py:transition_state():617] send defer: 1
2024-08-09 12:07:00,031 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:07:00,031 INFO    HandlerThread:527930 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-09 12:07:00,031 DEBUG   SenderThread:527930 [sender.py:send_request():409] send_request: defer
2024-08-09 12:07:00,031 INFO    SenderThread:527930 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-09 12:07:00,031 INFO    SenderThread:527930 [sender.py:transition_state():617] send defer: 2
2024-08-09 12:07:00,031 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:07:00,031 INFO    HandlerThread:527930 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-09 12:07:00,031 INFO    HandlerThread:527930 [system_monitor.py:finish():203] Stopping system monitor
2024-08-09 12:07:00,031 DEBUG   SystemMonitor:527930 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-09 12:07:00,031 INFO    HandlerThread:527930 [interfaces.py:finish():202] Joined cpu monitor
2024-08-09 12:07:00,031 DEBUG   SystemMonitor:527930 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-09 12:07:00,031 DEBUG   SystemMonitor:527930 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-09 12:07:00,032 INFO    HandlerThread:527930 [interfaces.py:finish():202] Joined disk monitor
2024-08-09 12:07:00,053 INFO    HandlerThread:527930 [interfaces.py:finish():202] Joined gpu monitor
2024-08-09 12:07:00,053 INFO    HandlerThread:527930 [interfaces.py:finish():202] Joined memory monitor
2024-08-09 12:07:00,053 INFO    HandlerThread:527930 [interfaces.py:finish():202] Joined network monitor
2024-08-09 12:07:00,053 DEBUG   SenderThread:527930 [sender.py:send_request():409] send_request: defer
2024-08-09 12:07:00,053 INFO    SenderThread:527930 [sender.py:send_request_defer():613] handle sender defer: 2
2024-08-09 12:07:00,053 INFO    SenderThread:527930 [sender.py:transition_state():617] send defer: 3
2024-08-09 12:07:00,053 DEBUG   SenderThread:527930 [sender.py:send():382] send: stats
2024-08-09 12:07:00,053 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:07:00,053 INFO    HandlerThread:527930 [handler.py:handle_request_defer():172] handle defer: 3
2024-08-09 12:07:00,053 DEBUG   SenderThread:527930 [sender.py:send_request():409] send_request: defer
2024-08-09 12:07:00,053 INFO    SenderThread:527930 [sender.py:send_request_defer():613] handle sender defer: 3
2024-08-09 12:07:00,053 INFO    SenderThread:527930 [sender.py:transition_state():617] send defer: 4
2024-08-09 12:07:00,054 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:07:00,054 INFO    HandlerThread:527930 [handler.py:handle_request_defer():172] handle defer: 4
2024-08-09 12:07:00,054 DEBUG   SenderThread:527930 [sender.py:send_request():409] send_request: defer
2024-08-09 12:07:00,054 INFO    SenderThread:527930 [sender.py:send_request_defer():613] handle sender defer: 4
2024-08-09 12:07:00,054 INFO    SenderThread:527930 [sender.py:transition_state():617] send defer: 5
2024-08-09 12:07:00,054 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:07:00,054 INFO    HandlerThread:527930 [handler.py:handle_request_defer():172] handle defer: 5
2024-08-09 12:07:00,054 DEBUG   SenderThread:527930 [sender.py:send():382] send: summary
2024-08-09 12:07:00,054 INFO    SenderThread:527930 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-09 12:07:00,054 DEBUG   SenderThread:527930 [sender.py:send_request():409] send_request: defer
2024-08-09 12:07:00,054 INFO    SenderThread:527930 [sender.py:send_request_defer():613] handle sender defer: 5
2024-08-09 12:07:00,054 INFO    SenderThread:527930 [sender.py:transition_state():617] send defer: 6
2024-08-09 12:07:00,054 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:07:00,054 INFO    HandlerThread:527930 [handler.py:handle_request_defer():172] handle defer: 6
2024-08-09 12:07:00,054 DEBUG   SenderThread:527930 [sender.py:send_request():409] send_request: defer
2024-08-09 12:07:00,054 INFO    SenderThread:527930 [sender.py:send_request_defer():613] handle sender defer: 6
2024-08-09 12:07:00,055 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: status_report
2024-08-09 12:07:00,075 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773
2024-08-09 12:07:00,075 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:00,075 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3D2FDEE9E72</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:00,152 INFO    SenderThread:527930 [sender.py:transition_state():617] send defer: 7
2024-08-09 12:07:00,152 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:07:00,153 INFO    HandlerThread:527930 [handler.py:handle_request_defer():172] handle defer: 7
2024-08-09 12:07:00,153 DEBUG   SenderThread:527930 [sender.py:send_request():409] send_request: defer
2024-08-09 12:07:00,153 INFO    SenderThread:527930 [sender.py:send_request_defer():613] handle sender defer: 7
2024-08-09 12:07:00,413 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f
2024-08-09 12:07:00,413 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294203', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:00,413 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3D300EB280C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:00,888 INFO    Thread-12 :527930 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/files/config.yaml
2024-08-09 12:07:00,888 INFO    Thread-12 :527930 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/files/output.log
2024-08-09 12:07:00,889 INFO    Thread-12 :527930 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/files/wandb-summary.json
2024-08-09 12:07:01,030 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-09 12:07:01,232 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2
2024-08-09 12:07:01,233 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5251', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:01,233 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3D346D7825B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:01,363 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773
2024-08-09 12:07:01,363 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:01,363 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3D34E8D5E0F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:01,363 INFO    wandb-upload_1:527930 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-09 12:07:01,640 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f
2024-08-09 12:07:01,640 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294203', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:01,640 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3D35B9911AC</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:01,989 INFO    SenderThread:527930 [sender.py:transition_state():617] send defer: 8
2024-08-09 12:07:01,990 DEBUG   SenderThread:527930 [sender.py:send_request():409] send_request: poll_exit
2024-08-09 12:07:01,990 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:07:01,990 INFO    HandlerThread:527930 [handler.py:handle_request_defer():172] handle defer: 8
2024-08-09 12:07:01,990 DEBUG   SenderThread:527930 [sender.py:send_request():409] send_request: defer
2024-08-09 12:07:01,990 INFO    SenderThread:527930 [sender.py:send_request_defer():613] handle sender defer: 8
2024-08-09 12:07:01,990 INFO    SenderThread:527930 [job_builder.py:build():298] Attempting to build job artifact
2024-08-09 12:07:01,991 INFO    SenderThread:527930 [job_builder.py:_get_source_type():428] is repo sourced job
2024-08-09 12:07:02,006 INFO    SenderThread:527930 [job_builder.py:build():404] adding wandb-job metadata file
2024-08-09 12:07:02,017 INFO    SenderThread:527930 [sender.py:transition_state():617] send defer: 9
2024-08-09 12:07:02,018 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:07:02,018 DEBUG   SenderThread:527930 [sender.py:send():382] send: artifact
2024-08-09 12:07:02,018 INFO    HandlerThread:527930 [handler.py:handle_request_defer():172] handle defer: 9
2024-08-09 12:07:02,031 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-09 12:07:02,528 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77
2024-08-09 12:07:02,528 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:02,528 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3D3941B3653</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:02,631 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73
2024-08-09 12:07:02,631 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:02,631 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3D39671304F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:02,889 INFO    Thread-12 :527930 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120657-2lwarbf1/files/output.log
2024-08-09 12:07:03,498 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2
2024-08-09 12:07:03,498 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5251', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:03,498 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3D3CDE89304</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:03,654 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77
2024-08-09 12:07:03,654 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:03,654 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3D3D7377331</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:03,883 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73
2024-08-09 12:07:03,884 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:03,884 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3D3E3557777</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:03,897 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773
2024-08-09 12:07:03,897 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:03,897 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3D3E5AF002B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:04,138 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f
2024-08-09 12:07:04,138 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294203', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:04,138 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3D3EDF09481</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:05,797 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77
2024-08-09 12:07:05,797 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:05,798 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3D456D9C0EB</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:05,798 INFO    wandb-upload_3:527930 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-09 12:07:06,428 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73
2024-08-09 12:07:06,428 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:06,428 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3D47B1C367C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:06,428 INFO    wandb-upload_4:527930 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-09 12:07:07,034 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:07:08,147 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2
2024-08-09 12:07:08,147 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5251', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:08,148 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3D4E3061F33</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:08,500 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773
2024-08-09 12:07:08,500 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:08,500 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3D4F8078BBA</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:08,607 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f
2024-08-09 12:07:08,607 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294203', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:08,607 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3D4FBF72030</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:10,305 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77
2024-08-09 12:07:10,305 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:10,305 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3D56391B403</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:10,586 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73
2024-08-09 12:07:10,586 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:10,586 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3D5732E08CF</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:12,037 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:07:17,039 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:07:18,076 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f
2024-08-09 12:07:18,076 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294203', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:18,076 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3D730B576A2</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:18,174 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773
2024-08-09 12:07:18,174 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:18,174 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3D7356FE974</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:18,213 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2
2024-08-09 12:07:18,213 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5251', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:18,213 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3D73B097369</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:19,093 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77
2024-08-09 12:07:19,094 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:19,094 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3D76F64EC17</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:20,301 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73
2024-08-09 12:07:20,301 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:20,301 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3D7B60C52B8</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:22,042 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:07:27,045 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:07:32,047 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:07:34,862 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773
2024-08-09 12:07:34,862 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:34,862 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3DB191D4EC8</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:35,766 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77
2024-08-09 12:07:35,766 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:35,766 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3DB51563618</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:35,980 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2
2024-08-09 12:07:35,980 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5251', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:35,980 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3DB5DDE94D0</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:37,049 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:07:37,074 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f
2024-08-09 12:07:37,075 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294203', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:37,075 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3DB96A04512</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:39,049 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73
2024-08-09 12:07:39,049 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:07:39,049 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3DC11BC9CEA</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:07:42,052 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:07:47,055 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:07:52,057 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:07:57,060 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:08:02,062 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:08:07,065 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:08:08,664 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a8bf533eab0d4da5ab4997772ca17ac1eb13b9471a8a83aee0ad3f2789d94773
2024-08-09 12:08:08,664 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:08:08,665 ERROR   wandb-upload_1:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3E2F532BB2A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:08:10,861 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=09d7fadf2b44d843ad137874cdcb9a296a5e54a81956bb1190e5f78951df96d2
2024-08-09 12:08:10,861 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5251', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:08:10,861 ERROR   wandb-upload_0:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3E37CDEC378</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:08:12,067 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:08:13,985 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2lwarbf1/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040659Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6b744bd76fdb3e8824e4dafd3dbd71281b154b244b641a9ae4ce64d8156e3e2f
2024-08-09 12:08:13,985 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294203', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:08:13,985 ERROR   wandb-upload_2:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2lwarbf1/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2lwarbf1/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3E427A064B1</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:08:14,418 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=afbd9833b5a0cff96a113d844db834640912af4a53e6297660de2fed18fb7e77
2024-08-09 12:08:14,418 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:08:14,418 ERROR   wandb-upload_3:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3E4510244C9</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:08:14,870 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040702Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=4a261bc5dd3f210343d3b574cf4856601873b07cdfacfc63363d2a088cc69d73
2024-08-09 12:08:14,870 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:08:14,870 ERROR   wandb-upload_4:527930 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/541/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3E468EF69B9</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:08:17,070 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:08:22,072 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:08:27,075 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:08:32,077 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:08:37,080 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:08:42,082 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:08:47,085 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:08:52,087 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:08:57,090 DEBUG   HandlerThread:527930 [handler.py:handle_request():146] handle_request: keepalive
